cookbook/sitemap-provider.rst

```diff
@@ -0,0 +1,88 @@
+Provider for XML-Sitemap
+========================
+
+`SitemapProvider` are used to load data for the XML-Sitemap. It returns
+an array of `SitemapUrl` instances. This Api has to be paginated because
+Google only allows `SitemapProviderInterface::PAGE_SIZE = 50000` urls in
```

Comment:
Wouldn't write the constant in here like that. maybe `Google only allows 50000 urls`? If you think it is necessary I would mention the constant in a separate sentence.